webapp/components/PasswordReset/Request.spec.js

```diff
@@ -26,12 +22,17 @@ describe('Request', () => {
       },
     }
     stubs = {
-      LocaleSwitch: "<div class='stub'></div>",
+      LocaleSwitch: true,
+      'sweetalert-icon': true,
+      'client-only': true,
+      'nuxt-link': true,
```

Comment:
before you used `NuxtLink: RouterLinkStub` whats the difference?

Reply:
I do not care